### PR TITLE
Results table -> Material Table

### PR DESCRIPTION
**Pull request by gordocanchola**
This PR is for using material-table for our results table.
This would replace the custom built results table
  
Material-table provides built-in functionality such as column reordering, sorting, search bar for filtering.

### CODE REVIEW

**Comment by willwill96** [comment]
Can you post a screenshot of what this looks like?

**Comment by gordocanchola** [comment]
image

@willwill96 
Working on getting the keyboard interaction with the selection interface working like it did on the old table. 
then I'll look into checkbox column size.

File: src/main/webapp/components/results/results.stories.js [hunk 1]
```diff
@@ -7,13 +7,13 @@ import genResults from '../../gen-results'
...
-  'Modified',
-  'Checksum',
-  'ID',
+  'title',
+  'thumbnail',
+  'description',
+  'created',
+  'modified',
+  'checksum',
+  //  'id',
```
Comment (AzGoalie) [inline src/main/webapp/components/results/results.stories.js]: Why is id commented out?

Comment (gordocanchola) [reply src/main/webapp/components/results/results.stories.js]: I commented that out for testing. Wasn't supposed to commit that.
Good catch. Thank you

Comment (gordocanchola) [reply src/main/webapp/components/results/results.stories.js]: fixed

**Pull request merged**